README.md

```diff
@@ -170,10 +174,24 @@ determining that location is as follows:
 -   [gcloud sdk](https://cloud.google.com/sdk/install) >= 269.0.0
 -   [jq](https://stedolan.github.io/jq/) >= 1.6
 -   [Terraform](https://www.terraform.io/downloads.html) >= 0.12.6
--   [terraform-provider-google] plugin >= 2.1, < 4.0
--   [terraform-provider-google-beta] plugin >= 2.1, < 4.0
+-   [terraform-provider-google] plugin >= 3.1, < 4.0
+-   [terraform-provider-google-beta] plugin >= 3.1, < 4.0
 -   [terraform-provider-gsuite] plugin 0.1.x if GSuite functionality is desired
 
+#### `terraform-provider-google` version 2.x
```

Comment:
I added a note in the readme about no longer supporting google provider 2.x and to use the previous version of this module if you need 2.x

But what's on master hasn't been pushed to terraform yet. We should push a release of what's in master before merging this so there is a 2.x version to use

Reply:
Master was released as `6.2.1`. Updated readme to reflect